psse/psse-model/src/test/java/com/powsybl/psse/model/PsseRawReaderTest.java

```diff
@@ -33,4 +36,88 @@ public void ieee14BusTest() throws IOException {
+                "f3", "o4", "f4", "wmod", "wpf"};
+            String[] actualGeneratorDataReadFields = context.getGeneratorDataReadFields();
+            assertTrue(compareReadFields(expectedGeneratorDataReadFields, actualGeneratorDataReadFields));
+
+            String[] expectedNonTransformerBranchDataReadFields = new String[] {"i", "j", "ckt", "r", "x", "b",
+                "ratea", "rateb", "ratec", "gi", "bi", "gj", "bj", "st", "met", "len", "o1", "f1", "o2", "f2", "o3",
+                "f3", "o4", "f4"};
+            String[] actualNonTransformerBranchDataReadFields = context.getNonTransformerBranchDataReadFields();
+            assertTrue(compareReadFields(expectedNonTransformerBranchDataReadFields, actualNonTransformerBranchDataReadFields));
+
+            String[] expected2wTransformerDataReadFields = new String[] {"i", "j", "k", "ckt", "cw", "cz", "cm",
+                "mag1", "mag2", "nmetr", "name", "stat", "o1", "f1", "o2", "f2", "o3", "f3", "o4", "f4", "r12", "x12", "sbase12"};
+            String[] actual2wTransformerDataReadFields = context.get2wTransformerDataReadFields();
+            assertTrue(compareReadFields(expected2wTransformerDataReadFields, actual2wTransformerDataReadFields));
+
+            String[] expected2wTransformerW1DataReadFields = new String[] {"windv", "nomv", "ang", "rata", "ratb",
+                "ratc", "cod", "cont", "rma", "rmi", "vma", "vmi", "ntp", "tab", "cr", "cx"};
+            String[] actual2wTransformerW1DataReadFields = context.get2wTransformerDataWinding1ReadFields();
+            assertTrue(compareReadFields(expected2wTransformerW1DataReadFields, actual2wTransformerW1DataReadFields));
+
+            String[] expected2wTransformerW2DataReadFields = new String[] {"windv", "nomv"};
+            String[] actual2wTransformerW2DataReadFields = context.get2wTransformerDataWinding2ReadFields();
+            assertTrue(compareReadFields(expected2wTransformerW2DataReadFields, actual2wTransformerW2DataReadFields));
+
+            String[] expectedAreaInterchangeDataReadFields = new String[] {"i", "isw", "pdes", "ptol", "arname"};
+            String[] actualAreaInterchangeDataReadFields = context.getAreaInterchangeDataReadFields();
+            assertTrue(compareReadFields(expectedAreaInterchangeDataReadFields, actualAreaInterchangeDataReadFields));
+
+            String[] expectedZoneDataReadFields = new String[] {"i", "zoname"};
+            String[] actualZoneDataReadFields = context.getZoneDataReadFields();
+            assertTrue(compareReadFields(expectedZoneDataReadFields, actualZoneDataReadFields));
+
+            String[] expectedOwnerDataReadFields = new String[] {"i", "owname"};
+            String[] actualOwnerDataReadFields = context.getOwnerDataReadFields();
+            assertTrue(compareReadFields(expectedOwnerDataReadFields, actualOwnerDataReadFields));
+        }
+    }
+
+    private boolean compareReadFields(String[] expected, String[] actual) {
```

Comment:
Why don't you use AssertArrayEquals?
You can probably also use Arrays.equals if you want to use your logging method

Reply:
Done